src/main/java/de/tum/in/www1/artemis/repository/ModelClusterRepository.java

```diff
@@ -18,6 +18,13 @@
 @Repository
 public interface ModelClusterRepository extends JpaRepository<ModelCluster, Long> {
 
+    @Query("""
+            SELECT COUNT (DISTINCT cluster)
+            FROM ModelCluster cluster
+            WHERE
+            cluster.exercise.id = :#{#exerciseId}""")
```

Comment:
the formatting here could be improved
```suggestion
                cluster.exercise.id = :#{#exerciseId}
        """)
```